riff-raff/app/notification/hooks.scala

```diff
@@ -85,17 +85,18 @@ object HookConfig {
     HookConfig(UUID.randomUUID(), projectName, stage, url, enabled, new DateTime(), updatedBy)
 }
 
-object HooksClient extends LifecycleWithoutApp with Logging {
-  trait Event
-  case class Finished(uuid: UUID, params: DeployParameters)
-
+class HooksClient(wsClient: WSClient) extends Lifecycle with Logging {
   lazy val system = ActorSystem("notify")
   val actor = try {
-    Some(system.actorOf(Props[HooksClient], "hook-client"))
-  } catch { case t:Throwable => None }
+    Some(system.actorOf(Props(classOf[HooksClientActor], wsClient), "hook-client"))
```

Comment:
I'm happy to switch everything over if we conclude that is a good thing to do (I'll add it to tonights agenda 😉). In the meantime I'm going to merge this as it is.